daemons/pid/simple.py

```diff
@@ -32,12 +33,31 @@ def pid(self):
 
                 try:
 
-                    return int(pidfile.read().strip())
+                    pid = int(pidfile.read().strip())
 
                 except ValueError:
 
                     return None
 
+		try:
+
+		    os.kill(pid, 0)
+
+		except OSError as e:
+
+		    if e.errno == errno.EPERM:
+
+			return pid
```

Comment:
Ah, OK. That makes sense. I had to go back and read the interface documentation to remind myself that returning `None` was the case that resulted in another process. :+1: